test/unit/dom/dom.js

```diff
@@ -0,0 +1,51 @@
+suite('DOM', function(){
+  suite('p5.prototype.select', function() {
+    var select = p5.prototype.select;
+
+    suite('select()', function() {
+      var elt;
+
+      teardown(function() {
+        if (elt && elt.parentNode) {
+          elt.parentNode.removeChild(elt);
+          elt = null;
+        }
+      });
+
+      test('should find elements by ID', function() {
+        elt = document.createElement('div');
+        elt.setAttribute('id', 'blarg');
+        document.body.appendChild(elt);
```

Comment:
I considered just using p5.dom to do this work, but I decided against it, since it would make the tests less isolated (e.g. bugs in e.g. `remove()` would bleed into these tests). However, there are also arguments [against](http://c2.com/cgi/wiki?UnitTestIsolation) unit test isolation, so I'm curious what others think.